core/shared/src/main/scala/monocle/std/Tree.scala

```diff
@@ -15,19 +14,19 @@ trait TreeOptics {
   final def rootLabel[A]: Lens[Tree[A], A] =
     Lens[Tree[A], A](_.rootLabel)(l => tree => Node(l, tree.subForest))
 
-  final def subForest[A]: Lens[Tree[A], Stream[Tree[A]]] =
-    Lens[Tree[A], Stream[Tree[A]]](_.subForest)(children => tree => Node(tree.rootLabel, children))
+  final def subForest[A]: Lens[Tree[A], LazyList[Tree[A]]] =
```

Comment:
scalaz still seem to be using Stream, so I think we are stuck with it. I guess it is fine because we are dropping support for scalaz after the next release